website/content/en/docs/concepts/node-templates.md

```diff
@@ -160,12 +161,16 @@ spec:
 
 The AMI used when provisioning nodes can be controlled by the `amiFamily` field. Based on the value set for `amiFamily`, Karpenter will automatically query for the appropriate [EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) via AWS Systems Manager (SSM). When an `amiFamily` of `Custom` is chosen, then an `amiSelector` must be specified that informs Karpenter on which custom AMIs are to be used.
 
-Currently, Karpenter supports `amiFamily` values `AL2`, `Bottlerocket`, `Ubuntu` and `Custom`. GPUs are only supported with `AL2` and `Bottlerocket`. The `AL2` amiFamily does not support ARM64 GPU instance types unless you specify a custom amiSelector.
+Currently, Karpenter supports `amiFamily` values `AL2`, `Bottlerocket`, `Ubuntu`, `Windows2019`, `Windows2022` and `Custom`. GPUs are only supported with `AL2`, `Bottlerocket`, `Windows2019` and `Windows2022`. The `AL2` amiFamily does not support ARM64 GPU instance types unless you specify a custom amiSelector.
+
+{{% alert title="Defaults" color="secondary" %}}
+If no `amiFamily` is defined, Karpenter will set the default `amiFamily` to AL2
 
 ```yaml
 spec:
-  amiFamily: Bottlerocket
+  amiFamily: AL2
 ```
+{{% /alert %}}
 
 ## spec.amiSelector
```

Comment:
We should consider adding an example or a comment in `amiSelector` specifying how Windows users can use Full AMIs using this one. Example-
```
  amiSelector:
    aws::name: Windows_Server-2019-English-Full-EKS_Optimized-1.26*
```

Reply:
Thats a good suggestion. I will add such an example.

Reply:
Added a FAQ section which

1. Explains that Windows Server Core is the option used with the Windows2019 and Windows2022 AMI families.
2. Provides an example of how to use the full AMI family